python/pyspark/sql/context.py

```diff
@@ -553,34 +560,16 @@ def register(self, name, f, returnType=StringType()):
 def _test():
     import os
     import doctest
-    import tempfile
     from pyspark.context import SparkContext
     from pyspark.sql import Row, SQLContext
     import pyspark.sql.context
-
```

Comment:
Let's leave the newline here. Usually the block of imports has a newline (I believe pep8 will complain).